File: codingTest/baekjoon/python/17224.py
# N, L, K = map(int, input().split())
# total = 0
# arr = []
# for _ in range(N):
#     if K == 0:
#         break
#     sub1, sub2 = map(int, input().split())
#     if sub1 <= L and sub2 <= L:
#         total += 140
#         K -= 1
#     else:
#         arr.append(sub1)

# if K > 0:
#     for sub1 in sorted(arr):
#         if K > 0 and sub1 <= L:
#             K -= 1
#             total += 100
#         else:
#             break

# print(total)
N, L, K = map(int, input().split())
quizList = []
for _ in range(N): quizList.append(tuple(map(int, input().split())))
quizList.sort(key=lambda x: x[1])
totalScore = 0
for quiz in quizList:
    if K == 0: break
    if quiz[0] > L: continue
    if quiz[1] <= L: 
        K -= 1
        totalScore += 140
    elif quiz[0] <= L:
        K -= 1
        totalScore += 100
print(totalScore)

File: codingTest/baekjoon/javascript/17224.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

const input = [];
rl.on("line", line => {
  input.push(line);
}).on("close", () => {
  const [N, L, K] = input
    .shift()
    .split(" ")
    .map(e => +e);
  let easy = 0,
    hard = 0;
  for (let i = 0; i < N; i++) {
    const [sub1, sub2] = input
      .shift()
      .split(" ")
      .map(e => +e);
    if (sub2 <= L) hard++;
    else if (sub1 <= L) easy++;
  }
  console.log(Math.min(hard, K) * 140 + (hard < K ? Math.min(K - hard, easy) * 100 : 0));
});
